imports/client/ui/components/timeLine/TimeLine.jsx

```diff
@@ -42,64 +42,65 @@ export default class TimeLine extends React.Component {
 
   openMaxDatePicker = () => {
     this.refs.maxDatePicker.focus()
-    //console.log('maxDAte')
+
   }
-PauseOrClear = (event ) => {
 
+  pauseOrClear = (event ) => {
 
-   window.clearInterval(this.tid);
-  if (this.playOrResumeTrig == 1){
-    //if paused before :clean
-    if (this.PauseOrClearTrig ==1) {
-      k = Math.round(this.props.ui.minTime)
-      this.props.updateUI({currentSliderTime :  k })
-      this.PauseOrClearTrig= 0
-      this.playOrResumeTrig =0
-      }
-    else {
 
-      this.PauseOrClearTrig= 1
+     window.clearInterval(this.timerForPlay);
+    if (this.playOrResumeTrig == 1){
+      //if paused before :clean
+      if (this.pauseOrClearTrig ==1) {
+        k = Math.round(this.props.ui.minTime)
+        this.props.updateUI({currentSliderTime :  k })
+        this.pauseOrClearTrig= 0
```

Comment:
Also you don't want to user 0 and 1 in JS if what you mean is `true` and `false`. The more explicit the code looks, the better it is. 

With React `this.pauseOrClearTrig= 0` will look like `this.setState({ pauseOrClearTrig : false})`. Then you can fetch the value later by using `this.state.pauseOrClearTrig`.

It is better because it keeps things organized, so we know what it the current state, how it was assigned, etc.